vertx-web/src/main/java/io/vertx/ext/web/Router.java

```diff
@@ -361,6 +361,17 @@ static Router router(Vertx vertx) {
   Router exceptionHandler(@Nullable Handler<Throwable> exceptionHandler);
 
   /**
+   * If true then the normalised request path will be used when routing (e.g. removing duplicate /)
+   * by default for all routes on this router, unless explicitly set.
+   * Default is true
+   *
+   * @param useNormalisedPathByDefault  use normalised path by default for all routes on this router?
+   * @return a reference to this, so the API can be used fluently
+   */
+  @Fluent
+  Router useNormalisedPathByDefault(boolean useNormalisedPathByDefault);
```

Comment:
I added byDefault because useNormalisedPath may imply that it will override the behavior for all routes, and there should be a distinciton.